transport-native-kqueue/src/main/java/io/netty/channel/kqueue/KQueueEventLoop.java

```diff
@@ -15,8 +15,10 @@
  */
 package io.netty.channel.kqueue;
 
+import io.netty.channel.ChannelFactory;
 import io.netty.channel.EventLoop;
 import io.netty.channel.EventLoopGroup;
+import io.netty.channel.ReflectiveChannelFactory;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'io.netty.channel.ReflectiveChannelFactory'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)